src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,57 +70,173 @@ contract DssSpellCollateralAction {
+        require(GemJoinAbstract(MCD_JOIN_RWA007_A).dec()                             == decimals,                "join-dec-not-match");
+
+        require(RwaUrnLike(RWA007_A_URN).vat()                                       == MCD_VAT,                 "urn-vat-not-match");
+        require(RwaUrnLike(RWA007_A_URN).jug()                                       == MCD_JUG,                 "urn-jug-not-match");
+        require(RwaUrnLike(RWA007_A_URN).daiJoin()                                   == MCD_JOIN_DAI,            "urn-daijoin-not-match");
+        require(RwaUrnLike(RWA007_A_URN).gemJoin()                                   == MCD_JOIN_RWA007_A,       "urn-gemjoin-not-match");
+        require(RwaUrnLike(RWA007_A_URN).outputConduit()                             == RWA007_A_OUTPUT_CONDUIT, "urn-outputconduit-not-match");
+
+        require(RwaOutputConduitLike(RWA007_A_OUTPUT_CONDUIT).psm()                  == MCD_PSM_USDC_A,          "output-conduit-psm-not-match");
+        
+        require(RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_URN).psm()                == MCD_PSM_USDC_A,          "input-conduit-urn-psm-not-match");
+        require(RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_URN).to()                 == RWA007_A_URN,            "input-conduit-urn-to-not-match");
+
+        require(RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_JAR).psm()                == MCD_PSM_USDC_A,          "input-conduit-jar-psm-not-match");
+        require(RwaInputConduitLike(RWA007_A_INPUT_CONDUIT_JAR).to()                 == RWA007_A_JAR,            "input-conduit-har-to-not-match");
+
+
+        // Init the RwaLiquidationOracle
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(ilk, RWA007_A_INITIAL_PRICE, RWA007_DOC, RWA007_A_TAU);
+        (, address pip, , ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+
+        // Set price feed for RWA007
+        DssExecLib.setContract(MCD_SPOT, ilk, "pip", pip);
+
+        // Init RWA007 in Vat
+        Initializable(MCD_VAT).init(ilk);
+        // Init RWA007 in Jug
+        Initializable(MCD_JUG).init(ilk);
+
+        // Allow RWA007 Join to modify Vat registry
+        DssExecLib.authorize(MCD_VAT, MCD_JOIN_RWA007_A);
+
+        // 100m debt ceiling
+        DssExecLib.increaseIlkDebtCeiling(ilk, RWA007_A_LINE, /* _global = */ true);
+
+        // Set the stability fee
+        DssExecLib.setIlkStabilityFee(ilk, RWA007_A_RATE, /* _doDrip = */ false);
+
+        // Set collateralization ratio
+        DssExecLib.setIlkLiquidationRatio(ilk, RWA007_A_MAT);
+
+        // Poke the spotter to pull in a price
+        DssExecLib.updateCollateralPrice(ilk);
+
+        // Give the urn permissions on the join adapter
+        DssExecLib.authorize(MCD_JOIN_RWA007_A, RWA007_A_URN);
+
+        // MCD_PAUSE_PROXY and Monetalis permission on URN
+        RwaUrnLike(RWA007_A_URN).hope(address(this));
```

Comment:
is the pause proxy supposed to have access here?

Reply:
I think so, as we have `auth`ed methods like `file` and `kiss`

Reply:
checking [here](https://forum.makerdao.com/t/rwa007-mip65-monetalis-clydesdale-ces-domain-team-assessment/17787#setting-up-the-vault-11), I see:
> Sets DsPauseProxy as auth role for RwaUrn and all Conduit contracts
can you confirm that's the expected behaviour (setting the pause proxy as operator)?

Reply:
oh! right, got confused with `RWA008` pattern

Reply:
in the conduits, pause proxy is `mate` as fallback, or it's required as well?

Reply:
As fallback